program/c/src/oracle/util/test_avg.c

```diff
@@ -76,8 +73,8 @@ main( int     argc,
 # define N 512
 
   ctr = 0;
-  for( int i=0; i<10000000; i++ ) {
```

Comment:
There's another variable called `i` inside the loop, can you fix that for clarity. For example by renaming this `i` to `iter`